src/ui/molecules/card/index.jsx

```diff
@@ -6,7 +6,7 @@ export const Card = ({ result, children }) => {
     <li className="card">
       {logo && <img src={`${process.env.PUBLIC_URL}/logos/${logo}`} height="150" width="250" alt={alt || ""} />}
       <h2>
-        <a href={url}>{title}</a>
+        <a href={url} target="_blank" rel="noopener noreferrer" style={{ textDecoration: 'none' }} >{title} </a>
```

Comment:
I understand the reasoning here, but I would leave this up to the user's choice. People generally know to use `cmd/ctrl` click if they want to open a new tab. It is also an accessibility problem so if we wanted to do this, we would have to add copy to say that clicking the link will open it in a new tab.

I would also not turn off text decoration on links for many of the same reasons. If we _did_ want to, I would instead do this in the CSS and not inline. That could cause problems down the road if someone wants to change it as this will override any style anywhere else.

Reply:
Thanks for the feedback @schalkneethling! I mostly use `cmd/ctrl` to open up the pages in new tab, and I didn't think of the cons while adding the inline style. My bad!

I have used Adobe Express for creating the logos to download it in correct pixel (500px x 300px). Since the file size of the logos are ~20kb, I didn't optimize it further.

I'm new to this and I'm trying to learn from it. Thank you!